<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>igwAnimationTimeWidget</class>
 <widget class="QWidget" name="igwAnimationTimeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>46</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="timeLabel">
     <property name="text">
      <string>Time:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="timeValueComboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="timestepCountLabel">
     <property name="text">
      <string>max is N</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="timestepValue">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
